internal/helper/yawollet.go

```diff
@@ -344,17 +369,17 @@ func createEnvoyTCPListener(
 			},
 		},
 		FilterChains: []*envoylistener.FilterChain{{
-			// proxy filter has to be the last in the chain
 			Filters: append(filters, &envoylistener.Filter{
+				// proxy filter has to be the last in the chain
 				Name: envoywellknown.TCPProxy,
 				ConfigType: &envoylistener.Filter_TypedConfig{
 					TypedConfig: listenPort,
 				},
 			}),
 		}},
-		Freebind:                      &wrappers.BoolValue{Value: true},
-		EnableReusePort:               &wrappers.BoolValue{Value: true},
-		PerConnectionBufferLimitBytes: &wrappers.UInt32Value{Value: 32768}, // 32 Kib
+		Freebind:                      wrapperspb.Bool(true),
+		EnableReusePort:               wrapperspb.Bool(true),
+		PerConnectionBufferLimitBytes: wrapperspb.UInt32(32768), // 32 Kib
```

Comment:
You can drop the comment, if you use `32 << 10` or `32 * 1024` here. Then it is clear enough without the comment and doesn't need extra maintenance once you want to change the value.